Portfolio:Finish mobile version

arnoldnekemiah opened the pull request:
-Added "about me" and "Contact" sections as required by the activity.

- Used CSS grid to place elements in the two sections as required by the activity.

-  Updated branch name from "added-finalchanges" to "portfolio-feature" as suggested by the previous reviewer.

Commit:
made final changes

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div>
+          <div class="about-img">
+            <img src="/Images/Img Placeholder.png" alt="">
+          </div>
```
- [ ] You are doing amazing work here but I thing you need to rely on the Figma design and properties for your styles so that you can match the requirement and make your portfolio page look really aesthetic for user experience and interface.
- Kindly check the comparison below and improve your design using correct `css styles` to apply `margin` or `padding`  where necessary.

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/220351674-ce9ab109-edee-4a68-81ce-3bca129220b0.png)|![image](https://user-images.githubusercontent.com/99715640/220351781-bd1c136d-6ada-4930-8769-7260dd400dd1.png)|

- The image on your project card is not showing here
- The `work section` is a lacking `padding` on both sides to move your content away from the extreme edges, especially from the left-hand side.
- `My Recent Work` is missing a lot of styles and please consider changing the `div` tag to a `hx` tag like `h2`,  `h3`....`h6` for semantic reasons. It will also boost user's accessibility on your page.
_Please check the figma design and try to make changes_  💪 💪

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+          <div>
+            <p class="Multi-Post-Heading">
+              Multi-Post Stories
+            </p>
```
- [ ] I think this is a title and a `hx` tag will be better and semantically correct for accessibility, and as a convention. Kindly change this to a title tag between `h2` - `h6` 👍

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [ ] I think this is a title and a `hx` tag will be better and semantically correct for accessibility, and as a convention. Kindly change this to a title tag between `h2` - `h6` 👍

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [ ] I think this is a title and a `hx` tag will be better and semantically correct for accessibility, and as a convention. Kindly change this to a title tag between `h2` - `h6` 👍

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [ ] I think this is a title and a `hx` tag will be better and semantically correct for accessibility, and as a convention. Kindly change this to a title tag between `h2` - `h6` 👍

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [x] I think this is a title and a `hx` tag will be better and semantically correct for accessibility, and as a convention. Kindly change this to a title tag between `h2` - `h6` 👍

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [x] I think this is a title and a `hx` tag will be better and semantically correct for accessibility, and as a convention. Kindly change this to a title tag between `h2` - `h6` 👍

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [ ] I think this is a title and a `hx` tag will be better and semantically correct for accessibility, and as a convention. Kindly change this to a title tag between `h2` - `h6` 👍

AbayomiOlaoye commented on style.css:
```diff
@@ -80,3 +80,245 @@ ul {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 20px;
+}
```
- [ ] This might be the main reason why you are having an overflow on the horizontal axis of your code. Remember that your coding  style was mobile-first? So, there is no way `3 columns` as declared here can conveniently display on a screen size of max-width 375px.

See the scroll bar:
|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/220357221-aebe30d0-4279-4e6c-8bce-1721fd9e4ea9.png)|![image](https://user-images.githubusercontent.com/99715640/220364137-538349af-6a61-4bbe-a7a8-6531b07ec8eb.png)|

- [ ] Kindly set the column to just 1 for a gouud design and user experience

AbayomiOlaoye commented on style.css:
```diff
@@ -80,3 +80,245 @@ ul {
+  grid-gap: 20px;
+}
+
+.art-container {
+  background-image: url("/Images/Img\ Placeholder\ \(1\).png");
+}
```
- I can see that you tried loading the images from here but as you can see above, no background image is loading up at the moment.
- [ ] Kindly check again, especially the file name or the file paths to be sure everything works correctly. This will make your page more interactive and improved user experience will be attained. 🐐

AbayomiOlaoye commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+          </div>
+          <div class="footer-img">
+            <img src="/Images/illustration-about-me-mobile.png" alt="img">
+          </div>
```
- [ ] Although this image is not showing now, it is not a major content for the page. It's just an aesthetic design for the background. Images like this are best added via the stylesheet using `background-image` properties. Kindly remove this and upload using your CSS just like you attempted for the project cards, and please try to make the image display properly. Your visitors will like it and appreciate your great design skills. 😉

AbayomiOlaoye commented on style.css:
```diff
@@ -80,3 +80,245 @@ ul {
+.button:disabled {
+  background-color: #f4f1f0;
+  color: #5b5957;
+}
```
- [ ] You have really done well working on the hover state of your button but it appears the `active` state is missing so that the color changes when user clicks the button. Kindly check the figma recommendation for button interaction. This goes a long way to boost user experience 🤗 

Just like this:
```
.button:active {
background: #E05304;
}
```

AbayomiOlaoye reviewed (changes requested):
## STATUS: CHANGE REQUIRED ♻️ 

Hi @arnoldnekemiah 👋 , 

Good job with this milestone so far!
There are just a few more issues to work on before moving on to the next project but you are almost there!

![so_close](https://user-images.githubusercontent.com/99715640/211950260-0e3881c0-9d4f-4ea2-a670-9ef45a23a0f5.gif)

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
used css as suggested by the reviewer

arnoldnekemiah commented:
I have made the changes as suggested by the reviewer.

aamirkhan2478 commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [ ] Kindly change this `p` tag to the `h2` tag it will be better and semantically correct for accessibility as mentioned by the previous reviewer. :point_up:  but still it's not changed so kindly fix this issue. :+1:

aamirkhan2478 commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+          <a href="#6" class="profesional-Art-btm button">See Project</a>
+        </div>
+        <div class="art-container">
+          <p class="profesional-Art-heading">
+            Profesional Art Printing Data
+          </p>
```
- [ ] As mentioned by the previous reviewer please use the `p` tag to the `h2` tag it will be better and semantically correct for accessibility as mentioned by the previous reviewer. :point_up: but still it's not changed so kindly fix this issue. :+1:

aamirkhan2478 commented on style.css:
```diff
@@ -80,3 +80,245 @@ ul {
+}
+
+/* work section */
+.work-container {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  grid-gap: 20px;
+}
```
- [ ] As mentioned by the previous reviewer kindly set the column to just 1 for a good design and user experience please check below :point_down:  my suggestions:

```suggestion
.work-container {
  display: grid;
  grid-template-columns: repeat(1, 1fr);
  grid-gap: 20px;
}
```

aamirkhan2478 commented on index.html:
```diff
@@ -43,6 +43,194 @@ <h2>I’m a Software Developer</h2>
+            <a href="#" class="footer-btm button">Get My Resume</a>
+          </div>
+          <div class="footer-img">
+            <img src="/Images/illustration-about-me-mobile.png" alt="img">
```
- [ ] As mentioned by the previous reviewer kindly use remove this `img` tag and use the `div` tag then give some class name and then use that class name in your CSS file and set your image inside your CSS file :+1:  check my suggestion below :point_down: 

 ```suggestion
            <div class="about-image"></div>
```

**Check below :point_down:  my style suggestions**

```
.about-image{
   background-image: url('./Images/illustration-about-me-mobile.png');
  background-repeat: no-repeat;
}
```

aamirkhan2478 reviewed (changes requested):
Hi @arnoldnekemiah , 

![](https://media4.giphy.com/media/CoND5j6Bn1QZUgm1xX/giphy.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.
✔️ All linters checked

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

arnoldnekemiah committed:
Update readme file

Commit:
worked on areas suggested by the previous 2 reviewers

Commit:
cleared stylelint error

Commit:
cleared stylelinter error

arnoldnekemiah commented:
-Replaced `<p` tag with `<h3` tag is specific areas as suggested by the reviewer

- Removed the img tag and replaced it with div tag and positioned the vector as required.

Awaiting aproval

Abenezer-Tilahun reviewed (approved):
Hi @arnoldnekemiah 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
